actors/abi/primitives.go

```diff
@@ -46,12 +50,15 @@ type Bytes []byte
 // TokenAmount is an amount of Filecoin tokens. This type is used within
 // the VM in message execution, to account movement of tokens, payment
 // of VM gas, and more.
-type TokenAmount int64 // TODO bigint
+type TokenAmount = big.Int
```

Comment:
Let's document this decision.
```
BigInt types are aliases rather than new types because the latter introduce incredible amounts of noise converting to and from types in order to manipulate values. We give up some type safety for ergonomics.
```